Source/WarpX.H

```diff
@@ -1535,6 +1535,8 @@ private:
 
     std::string restart_chkfile;
 
+    bool write_diagonstics_on_restart = false;
```

Comment:
@dpgrote please add a doxygen string for this member variable.
```suggestion
    /** .... */
    bool write_diagonstics_on_restart = false;
```